LBHackney-IT/lbh-webmap-template
# Task/add box to geoserver metadata

# Description

showTitleOnMobile added as an option on mobile

# How to test it

Test the CA map where the showTitleOnMobile is false at the moment. Change it to '"showTitleOnMobile": true, ' and check if it changes.
